File: DoAnOOP/DocGia.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DoAnOOP
{
    public class DocGia
    {
        public string MSDG { get; set; }
        public string HoTen { get; set; }
        public string DiaChi { get; set; }

        public DateTime NgaySinh { get; set; }
        public bool GioiTinh { get; set; }
        public string Email { get; set; }

        public DocGia(string MSDG, string HoTen, string DiaChi, DateTime NgaySinh, bool GioiTinh, string Email)
        {
            this.MSDG = MSDG;
            this.HoTen = HoTen;
            this.DiaChi = DiaChi;
            this.NgaySinh = NgaySinh;
            this.GioiTinh = GioiTinh;
            this.Email = Email;
        }
        public DocGia(DocGia dg)
        {
            this.MSDG = dg.MSDG;
            this.HoTen = dg.HoTen;
            this.DiaChi = dg.DiaChi;
            this.NgaySinh = dg.NgaySinh;
            this.GioiTinh = dg.GioiTinh;
            this.Email = dg.Email;
        }

    }
}

File: java/cuoiki/src/mypack/DOCGIA.java
package mypack;

import java.sql.Date;

public class DocGia {

	   public String MSDG;
       public String HoTen;
       public String DiaChi;

       public Date NgaySinh;
       public boolean GioiTinh;
       public String Email;
       
       public DocGia(String MSDG, String HoTen, String DiaChi, Date NgaySinh, boolean GioiTinh, String Email)
       {
           this.MSDG = MSDG;
           this.HoTen = HoTen;
           this.DiaChi = DiaChi;
           this.NgaySinh = NgaySinh;
           this.GioiTinh = GioiTinh;
           this.Email = Email;
       }
       public DocGia(DocGia dg)
       {
           this.MSDG = dg.MSDG;
           this.HoTen = dg.HoTen;
           this.DiaChi = dg.DiaChi;
           this.NgaySinh = dg.NgaySinh;
           this.GioiTinh = dg.GioiTinh;
           this.Email = dg.Email;
       }
       
       
       
	public String getMSDG() {
		return MSDG;
	}
	public void setMSDG(String mSDG) {
		MSDG = mSDG;
	}
	public String getHoTen() {
		return HoTen;
	}
	public void setHoTen(String hoTen) {
		HoTen = hoTen;
	}
	public String getDiaChi() {
		return DiaChi;
	}
	public void setDiaChi(String diaChi) {
		DiaChi = diaChi;
	}
	public Date getNgaySinh() {
		return NgaySinh;
	}
	public void setNgaySinh(Date ngaySinh) {
		NgaySinh = ngaySinh;
	}
	public boolean isGioiTinh() {
		return GioiTinh;
	}
	public void setGioiTinh(boolean gioiTinh) {
		GioiTinh = gioiTinh;
	}
	public String getEmail() {
		return Email;
	}
	public void setEmail(String email) {
		Email = email;
	}
       
       
       
}
